src/ledger/parse/payment.ts

```diff
@@ -27,12 +28,22 @@ function parsePayment(tx: any): Object {
     tag: tx.SourceTag
   }
 
-  const destination = {
+  const destination: {
+    address: string,
+    amount?: Amount,
+    amountInfo?: string,
+    tag: number | undefined
+  } = {
     address: tx.Destination,
-    amount: removeGenericCounterparty(parseAmount(tx.Amount), tx.Destination),
     tag: tx.DestinationTag
   }
 
+  // Omit 'amount' for partial payments to prevent misuse
+  if (tx.Amount && !utils.isPartialPayment(tx)) {
```

Comment:
What do you think about including the partial payment amount if the full amount was actually delivered?

Reply:
I would prefer that users instead look at deliveredAmount, always.

I've been considering going the route of actually omitting `amount` in *all* cases... 🤔